TACC/Core-Portal
# Bug/WP-50: Fix sizing of buttons "as-link"

## Overview

There was a bug that caused the "as-link" buttons to appear smaller than other buttons (0.78 rem as opposed to 1 rem).

## Related

* [WP-50](https://tacc-main.atlassian.net/browse/WP-50)

## Changes

After inspecting the as-link button in question, I navigated to the file that set the font-size to 0.78 rem and adjusted it to 1 rem.

## Testing

1. Log in to [cep.test/workbench](cep.test/workbench).
2. Navigate to the History section in the sidebar.
3. In both Jobs and Historic Jobs, the "Mark All as Viewed" as-link button should now appear bigger (16px).

## UI

![image](https://github.com/user-attachments/assets/189931c9-dc0d-43b8-87f4-81af592e5ea9)

## Notes

I am unable to confirm if the as-link button in the Shared Workspaces section was fixed as I do not have a Shared Workspace. Please navigate to this section to confirm whether or not my change was also applied there.
